phalcon/assets/manager.zep

```diff
@@ -488,6 +488,16 @@ class Manager
 					let prefixedPath = path;
 				}
 
+                let version = collection->getVersion();
+				if $resource->getVersion() == null && version != null {
+					if version {
+						if version === true && local {
+							let version = filemtime($resource->getRealSourcePath());
+						}
+						let prefixedPath = prefixedPath . "?ver=" . version;
+					}
+				}
```

Comment:
Well yea, cuz this will require to https://github.com/phalcon/cphalcon/pull/12591/files#diff-f14ebcff6721de771f55f77e6f962321R96 have another argument too.